gitpod-io/gitpod
# [server] remove /tos endpoint – WEB-121

This removes the obsolete ToS parts from server, while trying not to touch the current behavior. 

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - at-rm-tos</li>
	<li><b>🔗 URL</b> - <a href="https://at-rm-tos.preview.gitpod-dev.com/workspaces" target="_blank">at-rm-tos.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Related Issue(s)

Part of WEB-121

## How to test
1. Check that sign-in, log-out, and log-in does work as expected. 
2. Check that Git Integrations do work, e.g. updating scopes.

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
